### PR TITLE
Add documentation for the Twinkly integration

### PR DESCRIPTION
**Pull request by dr1rrb**
## Proposed change

Add documentation for the new Twinkly integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/42103
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1954
- This PR fixes or closes issue: _none_

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by dr1rrb** [comment]
@frenck @hunterjm If you don't any comment about the content of the doc, as the PR in core has been merged, I think this should be merged to 😊

**Comment by frenck** [comment]
Please don't push or ping for those things. It will be handled and picked up automatically. Thanks 👍

File: source/_integrations/twinkly.markdown [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+description: Instructions on how to integrate Twinkly led string to Home Assistant.
+ha_category:
+  - Light
+ha_release: '0.117'
+ha_config_flow: true
+ha_domain: twinkly
+ha_iot_class: Local pull
+---
+
+The `twinkly` integration allows you to control [Twinkly](https://twinkly.com/) LED string from Home Assistant.
```
Comment (frenck) [inline source/_integrations/twinkly.markdown]:
```suggestion
The Twinkly integration allows you to control [Twinkly](https://twinkly.com/) LED string from Home Assistant.
```

File: source/_integrations/twinkly.markdown [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+The Twinkly devices does not store the effects locally, they are instead re-sent from the Twinkly application each time.
+This means that this integration does not support to change the LED string effect.
+It only supports to configure the brightness and to turn the device on and off.
+
+## Setup
```
Comment (frenck) [inline source/_integrations/twinkly.markdown]:
```suggestion
## Configuration
```

File: source/_integrations/twinkly.markdown [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+You can setup this integration from the Home Assistant user interface,
```
Comment (frenck) [inline source/_integrations/twinkly.markdown]: We should describe here how to do that, e.g., by providing a click path and/or step by step instructions.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/twinkly.markdown [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+title: Twinkly
+description: Instructions on how to integrate Twinkly led string to Home Assistant.
+ha_category:
+  - Light
+ha_release: '0.117'
```
Comment (frenck) [inline source/_integrations/twinkly.markdown]:
```suggestion
ha_release: 0.119
```

File: source/_integrations/twinkly.markdown [hunk 5]
```diff
@@ -0,0 +1,22 @@
+---
+title: Twinkly
+description: Instructions on how to integrate Twinkly led string to Home Assistant.
```
Comment (frenck) [inline source/_integrations/twinkly.markdown]:
```suggestion
description: Instructions on how to integrate Twinkly LED string to Home Assistant.
```

**Review by frenck — APPROVED** [review]
Thanks! 👍

**Pull request merged**